AUTHORS.rst

```diff
@@ -5,9 +5,10 @@ Credits
 Development Lead
 ----------------
 
-* John Kirkham <kirkhamj@janelia.hhmi.org>
+* `@jakirkham <https://github.com/jakirkham>`_
```

Comment:
```suggestion
* John Kirkham `@jakirkham <https://github.com/jakirkham>`_
```

Reply:
You've lost John's full name with this edit, we want to put that back again

Reply:
Can you label this PR as `hacktoberfest-accepted`? Thanks.